lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_bump_version_hotfix.rb

```diff
@@ -43,19 +43,19 @@ def self.available_options
             key: :version,
             env_name: 'FL_IOS_BUMP_VERSION_HOTFIX_VERSION',
             description: 'The version of the hotfix',
-            is_string: true
+            type: String
           ),
           FastlaneCore::ConfigItem.new(
             key: :previous_version,
             env_name: 'FL_IOS_BUMP_VERSION_HOTFIX_PREVIOUS_VERSION',
             description: 'The version to branch from',
-            is_string: true
+            type: String
           ),
           FastlaneCore::ConfigItem.new(
             key: :skip_deliver,
             env_name: 'FL_IOS_BUMP_VERSION_HOTFIX_SKIP_DELIVER',
             description: 'Skips Deliverfile key update',
-            is_string: false, # Boolean parameter
+            type: Boolean,
```

Comment:
Side note: I think this `ConfigItem` has been removed by Ian as part of his trial project in `trial/iangmaia`, so this might create a conflict with his branch down the line.

I wonder if we might not be better off finishing [the migration document that he started to document breaking changes from his branch](https://github.com/wordpress-mobile/release-toolkit/pull/452/files), then merging his branch at least, to avoid that branch staying open too long and drifting away and getting lots of conflicts down the line—the more we wait to merge it, the more conflicts we'll risk having with subsequent requests.

This is obviously outside of this PR's scope, but just wanted to drop a note about it here in case you might have time to tackle that one next before moving to other work on the release-toolkit. Including because part of his work in his branch also contains updated or additional unit tests for some actions, and if that's also what you planned to worked on next, might be better to land theirs first too…